### PR TITLE
[ci] Try enabling MSVC and check build times

### PR DESCRIPTION
**Pull request by bobcao3**
Issue: #

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -344,7 +344,7 @@ jobs:
           - os: windows-2019
             llvmVer : '15'
             archs: "cpu,cuda,opengl"
-            runsOn: [self-hosted, windows, cuda, OpenGL]
+            runsOn: [self-hosted, windows, cuda, OpenGL, msvc-debug]
```
Comment (feisuzhu) [inline .github/workflows/testing.yml:347]: Should restore it

File: .github/workflows/scripts/win_build.ps1 [hunk 2]
```diff
@@ -69,7 +69,8 @@ if ($llvmVer -eq "10") {
     throw "Unsupported LLVM version"
 }
 
-$env:TAICHI_CMAKE_ARGS += " -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang"
+$env:TAICHI_USE_MSBUILD = 1
```
Comment (feisuzhu) [inline .github/workflows/scripts/win_build.ps1:72]: Where is it used? I can only see assignments but not reads.

**Review by feisuzhu — CHANGES_REQUESTED** [review]

File: .github/workflows/scripts/win_build.ps1 (hunk 2 above)
Comment (bobcao3) [reply .github/workflows/scripts/win_build.ps1:72]: It's a flag for `setup.py`

**Review by feisuzhu — APPROVED** [review]
LGTM

**Pull request merged**